src/class-shortcode.php

```diff
@@ -99,7 +99,7 @@ public static function render_results( $atts ) {
 					$query_args = array(
 						'posts_per_page'   => 10,
 						'post_type'        => 'result',
-						'post_parent'      => $rev_id,
+						'post_parent'      => $revision->post_name,
```

Comment:
The pull request says:

> Use post_name with the 'r'

But the commit is changing the `post_parent` argument?

`post_parent` needs to be/remain numeric.